obj2mjcf/_cli.py

```diff
@@ -102,6 +102,8 @@ class Args:
     vhacd_args: VhacdArgs = field(default_factory=VhacdArgs)
     """arguments to pass to V-HACD"""
     texture_args: TextureArgs = field(default_factory=TextureArgs)
+    overwrite: bool = False
+    """overwrite if existing files"""
```

Comment:
Could you change the description to "overwrite previous run output"?

Reply:
Of course, I have updated both the Readme and the docstring